### ISSUE TITLE
I have a dataset that is not in d4rl, how can I use it for training and testing?

### ISSUE THREAD

**Issue by ralgond** [CleanDiffuserTeam/CleanDiffuser#30]
我有一个不在d4rl数据集，请问如何才能使用它进行训练和测试？
I have a dataset that is not in d4rl, how can I use it for training and testing?

**Comment by pickxiguapi** [comment]
This is a good question, we will provide tutorials on dataset formats suitable for each algorithm within a week, until then you can try to debug the algorithms running on d4rl or robomimic to get the dataset format!

**Comment by ZibinDong** [comment]
I've been doing most of my updates on the lightning branch lately. I've rebuilt the entire codebase using PyTorch Lightning in this branch. It allows us to train models and use advanced deep learning techniques like parallel training and mixed precision in a much simpler way.

If you're working on developing new algorithms, I'd strongly recommend trying the lightning branch. I've also added some notebook tutorials to this branch to help you get up to speed with the PyTorch Lightning version of CleanDiffuser more quickly. These notebook tutorials also show how to train your models on your customized datasets.

If you're still not quite sure how to use your custom datasets after going through these notebook tutorials, or if you'd rather stick to training your models on the main branch, feel free to add me on WeChat: dongzibin1112. We can chat more about the code implementation there.
